gitlab/const.py

```diff
@@ -56,8 +54,18 @@
 ]
 
 
+class GitlabConstant:
+    def __new__(cls, value: Any) -> str:  # type: ignore
+        # This allows us to get a string representation of a value.
+        # For example: AccessLevel(10)
+        for attr, attr_value in cls.__dict__.items():
+            if value == attr_value:
+                return attr
+        raise ValueError(f"{value!r} is not a valid {cls.__qualname__}")
+
```

Comment:
I only put the `__new__()` method in so we could do `AccessLevel(10)`. But not sure if we need/desire that.